app/javascript/styles/mastodon/components.scss

```diff
@@ -5017,3 +5016,27 @@ noscript {
     left: 0;
   }
 }
+
+.floating-action-button {
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 3.9375rem;
+  height: 3.9375rem;
+  bottom: 1.3125rem;
+  right: 1.3125rem;
```

Comment:
@Gargron Why did you use rem which we haven't used before?

Reply:
I looked up how the floating action button was positioned in another Material UI application :thinking: I figured the button needs to be easily clickable so I shouldn't mess with these values.